Examples/ArcGISToolkitExamples/Misc/ARStatusViewController.swift

```diff
@@ -13,7 +13,7 @@
 
 import UIKit
 import ARKit
-
```

Comment:
I would suggest keeping the blank line between the import statements and the extension.

Reply:
Inadvertent deletion...